src/mame/mr/dribling.cpp

```diff
@@ -598,7 +598,26 @@ ROM_START( driblingo )
 	ROM_LOAD( "tbp24s10.2d",  0x0500, 0x0100, CRC(a17d6956) SHA1(81724daf2e2d319f55cc34cc881b6a9a4e64e7ac) )
 ROM_END
 
-ROM_START( driblingbr )
+// Original Model Racing PCB (EF00284)
+ROM_START( driblingam )
```

Comment:
Hang on, if it’s an original Model Racing PCB, How is it a bootleg?  If they needed to obtain original Model Racing PCBs to make it, how would the bootleg be any cheaper than the original?  That makes it sound more like a hack or a version licensed for Spanish distribution than a bootleg.

Are you sure it isn’t a licensed version of Spanish distribution, or an importer changing the ROMs for the local market?

Also, do you know whether this used original sound hardware, or the bootleggers changed it?

Reply:
Considering it’s apparently built with an original board, I’d go with either “(hack?)” or assuming it was licensed.  Using an original board to make a bootleg of the same game (as opposed to a bootleg conversion to a different game) just doesn’t make sense.

Reply:
Then I'd say it's a license, it does not make sense for a hack to present the machine on Italy on a trade fair referencing Model Racing (an Italian company). Neither make sense for a bootleg, indeed.
I'll change it tomorrow.
Thanks!

Reply:
Do we know if there is a different ROM?
Could this explain this [comment](https://github.com/mamedev/mame/pull/12127#issuecomment-1999902285)?

@cuavas OT: I had an idea about the PARATA JFET. How can I share/discuss it? Basically using the same AFUNC trick switching between current limiter and Ohmic region mode.

Reply:
> @cuavas OT: I had an idea about the PARATA JFET. How can I share/discuss it? Basically using the same AFUNC trick switching between current limiter and Ohmic region mode.

You could start a discussion over here, in the General or Ideas section: https://github.com/orgs/mamedev/discussions

Reply:
@cuavas  Thanks. I've actually made good progress with the first JFET behaving correctly. The sound comes out right but it's attenuated. Opening a PR hoping your help can get this done.